app/javascript/source/javascript/binx_admin.js

```diff
@@ -12,5 +14,29 @@ window.BinxAdmin = class BinxAdmin {
         .parents(".receptacle")
         .css("max-width", "100%");
     }
+    if (window.location.href.match('\\admin/graphs')){
+      this.initGraph()
+    };
+  }
+
+  initGraph() {
+    $("select#graph_date_option_choice").on("change", e => {
+      e.preventDefault()
+      if ($("select#graph_date_option_choice")[0].value === "custom") {
+        $("#calendar-box").slideDown()
+      }
+      else {
+        $("#calendar-box").slideUp()
+        let $this = $('#start_at')
```

Comment:
No reason to assign `$this` here, since you only use it once. You should call `.val` directly on `$("#calendar-box")`